src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -1511,6 +1279,236 @@ public double NextDouble()
             }
         }
 
+        #region Skypped tests.
+        [Fact(Skip = "DefineFunctions method is deprecated")]

```

Comment:
Do these run?

Reply:
These were changed to skip while the old code was not yet removed. These test cases are now removed.